### PR TITLE
Fix rendering of package information on the pkgdown website

### PR DESCRIPTION
**Pull request by cforgaci**
@fnattino, I noticed a few issues with the rendered pkgdown website: the image links on the landing page are broken and the Changelog page is empty.

I made the necessary changes following the pkgdown guidelines on [images/figures](https://pkgdown.r-lib.org/reference/build_home.html?q=images#images-and-figures) and [the news section](https://pkgdown.r-lib.org/reference/build_news.html). These fixes still fall under the release we just made, right?

### CODE REVIEW

**Review by fnattino — COMMENTED** [review]
Thank you so much @cforgaci! Well, this is actually the first PR after 0.1.0 has been released, so I would say let's add the first entry in NEWS.md under the `# Unreleased` section :) (and I think this should go under `## Fixed`?).

File: NEWS.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+# [Unreleased]
 
-## [0.1.0] - 2024-12-16
+# rcoins 0.1.0 - 2024-12-16
```
[the remark this answers is not in the document]
Comment (cforgaci) [reply NEWS.md]: I tried locally `# 0.1.0` and `# v0.1.0` too but I cannot build the news page that way. However, it works with `# Version 0.1.0`.

[the remark this answers is not in the document]
Comment (fnattino) [reply NEWS.md]: Ah weird, I was only wondering why - looks good to me!

**Review by fnattino — APPROVED** [review]
Perfect, thank you so much!

**Pull request merged**